src/SwedbankPay.Episerver.Checkout.OrderManagement/Steps/PaymentStep.cs

```diff
@@ -9,39 +9,40 @@
 using SwedbankPay.Sdk;
 
 using System.Linq;
+using System.Threading.Tasks;
 
 namespace SwedbankPay.Episerver.Checkout.OrderManagement.Steps
 {
     public abstract class PaymentStep
     {
         protected PaymentStep Successor;
-        protected PaymentMethodDto PaymentMethod { get; set; }
-        public ISwedbankPayClient SwedbankPayClient { get; }
-        public MarketId MarketId { get; }
 
         public PaymentStep(IPayment payment, IMarket market, ISwedbankPayClientFactory swedbankPayClientFactory)
         {
             MarketId = market.MarketId;
             PaymentMethod = PaymentManager.GetPaymentMethod(payment.PaymentMethodId);
-            
-            if(PaymentMethod != null)
-            {
+
+            if (PaymentMethod != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-episerver-checkout/pullRequest?prid=5328029)